### PR TITLE
refactor: refine some code and comments in the sema crate.

### PR DESCRIPTION
**Pull request by Peefy**
Purpose:
1. refactor some documents on the schema type in the sema crate.

### CODE REVIEW

File: kclvm/sema/src/ty/mod.rs [hunk 1]
```diff
@@ -151,14 +151,20 @@ pub struct SchemaType {
     pub filename: String,
     /// The schema definition document string.
     pub doc: String,
-    /// Mark whether the schema is a type of a instance or
-    /// a type value (Schema.instances()).
+    /// Indicates whether the schema is a type of a instance or
+    /// a type value.
+    /// ```no_check
+    /// schema Person:  `Person` is not a schema instance.
```
Comment (chai2010) [inline kclvm/sema/src/ty/mod.rs:157]: `Person` is not a schema instance.’ is comment, should begin with `#`

File: kclvm/sema/src/ty/mod.rs [hunk 2]
```diff
@@ -151,14 +151,20 @@ pub struct SchemaType {
...
     pub doc: String,
-    /// Mark whether the schema is a type of a instance or
-    /// a type value (Schema.instances()).
+    /// Indicates whether the schema is a type of a instance or
+    /// a type value.
+    /// ```no_check
+    /// schema Person:  # `Person` is not a schema instance.
+    ///     name: str
+    /// 
+    /// person = Person {name = "Alice"}  # `person` is a schema instance.
```
Comment (ldxdl) [inline kclvm/sema/src/ty/mod.rs:160]: add a sample case for a type value?

Comment (Peefy) [reply kclvm/sema/src/ty/mod.rs]: fixed.

**Review by chai2010 — APPROVED** [review]
LGTM

**Review by ldxdl — APPROVED** [review]
lgtm.

**Pull request merged**